### DISCUSSION TITLE
How can I configure task argument to use the path of the actual file edited?

### DISCUSSION THREAD

**Discussion by Tom-Boscher** [zed-industries/zed#8216] [Feature Requests]
I would like to create a task to run my Python script. Is it possible to set a dynamic path based on the file editing in the active pane?

**Comment by SomeoneToIgnore** [comment]
Not yet, dynamic features are being under development right now and so far [the task documentation](https://github.com/zed-industries/zed/blob/main/docs/src/tasks.md) explicitly states that static tasks are

> Tasks, defined in a config file (tasks.json in the Zed config directory) that do not depend on the current editor or its content.

[Last update](https://github.com/zed-industries/zed/issues/7108#issuecomment-1959999869) in the tracking issue covers future plans on adding more dynamic capabilities.
